### PR TITLE
add two custom certificates for x509 name parsing tests

### PR DESCRIPTION
**Pull request by reaperhulk**
needed for #1612

### CODE REVIEW

File: docs/development/test-vectors.rst [hunk 1]
```diff
@@ -92,6 +92,10 @@ Custom X.509 Vectors
   generated using OpenSSL.
 * ``ec_no_named_curve.pem`` - Contains an ECDSA certificate that does not have
   an embedded OID defining the curve.
+* ``all_supported_names.pem`` - An RSA 2048 bit certificate generated using
+  OpenSSL that contains a subject and issuer that have two of each supported
+  attribute type from :rfc:`5280`.
+* ``unsupported_subject_name.pem`` - An RSA 2048 bit self-signed CA certificate generated using OpenSSL that contains the unsupported "initials" name.
```
Comment (alex) [inline docs/development/test-vectors.rst]: This looks like it needs to be line-wrapped.

**Comment by alex** [comment]
inspecting vectors with `openssl x509` matches the description in the docs

**Comment by alex** [comment]
Also expected with some bespoke Go code to verify.

**Comment by alex** [comment]
inspected*

**Comment by alex** [comment]
retest this please

**Comment by reaperhulk** [comment]
Added one more certificate. Easiest way to test this is to use `openssl asn1parse -in <certpath>` to see that it is UTF8STRING and has the value "We heart UTF8!™"

openssl x509 -text will give you the ASCII bytes (way to go OpenSSL!).

**Comment by alex** [comment]
Confirmed with `openssl x509`, `openssl asn1parse`, and a bespoke go script.

**Pull request merged**